packages/www/components/WebhookDetails/index.tsx

```diff
@@ -125,4 +256,122 @@ const WebhookDetails = ({ id, data }) => {
   );
 };
 
+const Filters = ({ filters, activeFilter, handleFilterClick, logs }) => {
+  const totalWebhookLogs = logs?.length;
+
+  const totalSucceededWebhookLogs = logs?.filter(
+    (log) => log.response.status === 200
+  ).length;
+
+  const totalFailedWebhookLogs = logs?.filter(
+    (log) => log.response.status !== 200
+  ).length;
+
+  return (
+    <Flex
+      css={{
+        my: "$5",
+        gap: "$3",
+      }}>
+      {filters.map((filter) => (
+        <Box
+          onClick={() => handleFilterClick(filter)}
+          css={{
+            px: "$3",
+            py: "$2",
+            height: "100%",
+            border: activeFilter === filter ? "2px solid" : "1px solid",
+            borderColor: activeFilter === filter ? "$blue11" : "$neutral8",
+            width: "20%",
+            borderRadius: "$3",
```

Comment:
Let's add a `cursor: default` to this component otherwise the hover state on the text uses `cursor: text`